protocol/avdl/keybase1/simple_fs.avdl

```diff
@@ -281,13 +283,16 @@ protocol SimpleFS {
 
   /**
    Begin recursive copy of directory 
+
+   overwriteExistingFiles controls whether an existing file will be overwritten. It
```

Comment:
The previous behavior was equivalent to `overwriteExistingFiles = true`, right?  So this PR is changing the default behavior of the RPC I think.  It looks like the CLI copy command does a manual check for the file before it does the copy in order to enforce the overwrite flags.  So I guess this PR doesn't change the behavior from the CLI user's perspective, except in a weird race situation.  That's probably fine, I'm just mentioning it in case you didn't think through it, or if there are JS-side places that need updating for this new behavior that you haven't fixed yet.

Reply:
Yeah this change is intentional on my end. It doesn't seem great to automatically override files. Automatically renaming seems like the right behavior to me.

I actually didn't realize CLI did that manual check, and just assumed it'd make sense to error in CLI too. Thanks for checking it!